camera_info_right:
  height: 480
  width: 848
  camera_name: /camera/camera/infra2
  distortion_model: plumb_bob
  D: [-0.016400, 0.005199, -0.003613, -0.002662, 0.000000]
  K: [424.99857, 0., 413.16907, 0., 424.8757, 234.47621, 0., 0., 1.]
  R:
    [
      0.99995674,
      0.00223708,
      -0.00902811,
      -0.0023787,
      0.99987382,
      -0.01570608,
      0.00899184,
      0.01572687,
      0.99983589,
    ]
  P:
    [
      456.63488,
      0.,
      402.7873,
      1102.52914,
      0.,
      456.63488,
      246.32678,
      0.,
      0.,
      0.,
      1.,
      0.,
    ]
